workos/utils/request.py

```diff
@@ -48,7 +48,8 @@ def set_base_api_url(self, base_api_url):
     def generate_api_url(self, path):
         return self.base_api_url.format(path)
 
-    def build_parameterized_url(self, url, **params):
+    @classmethod
+    def build_parameterized_url(cls, url, **params):
```

Comment:
Still on the fence about the proper home for this method. Waiting until we're no longer RequestHelper dependent to move around.